library/src/main/scala/scala/scalajs/runtime/WasPublicBeforeTyper.scala

```diff
@@ -0,0 +1,5 @@
+package scala.scalajs.runtime
+
+import scala.annotation.StaticAnnotation
+
+class WasPublicBeforeTyper extends StaticAnnotation
```

Comment:
It is just because we did not have any other good locations to put it. It probably should be in its own library that is only used by the plugin, but this is a bit of overkill. Maybe in the future we could take out stuff that is only used by the plugin.